app/Providers/AppServiceProvider.php

```diff
@@ -206,6 +206,182 @@ function.
+                                'url' => route('exhibitions.upcoming'),
+                            ],
+                            [
+                                'name' => 'Archive',
+                                'url' => route('exhibitions.history'),
+                            ],
+                        ],
+                    ],
+                    [
+                        'name' => 'Art &amp; Artists',
+                        'description' => 'Explore the works in our collection and delve deeper into their stories.',
+                        'image' => url('/images/navigation-thumbnail-art-and-artists.jpg'),
+                        'children' => [
+                            [
+                                'name' => 'Artworks',
+                                'url' => route('collection'),
+                            ],
+                            [
+                                'name' => 'Articles &amp; Videos',
+                                'url' => route('articles_publications'),
+                            ],
+                            [
+                                'name' => 'Research',
+                                'url' => route('collection.research_resources'),
+                                'children' => [
+                                    [
+                                        'name' => 'Library',
+                                        'url' => '/library',
+                                    ],
+                                    [
+                                        'name' => 'Archival Collections',
+                                        'url' => '/archival-collections',
+                                    ],
+                                    [
+                                        'name' => 'Collection Information',
+                                        'url' => '/collection-information',
+                                    ],
+                                    [
+                                        'name' => 'Conservation and Science',
+                                        'url' => '/about-us/departments/conservation-and-science-2',
+                                    ],
+                                ],
+                            ],
+                            [
+                                'name' => 'Publications',
+                                'url' => route('articles_publications'),
+                                'children' => [
+                                    [
+                                        'name' => 'Print Catalogues',
+                                        'url' => route('collection.publications.printed-publications'),
+                                    ],
+                                    [
+                                        'name' => 'Digital Publications',
+                                        'url' => route('collection.publications.digital-publications'),
+                                    ],
```

Comment:
Figma has a third option for "Digital Backlog". I'm not sure where that would link to, but add the text here for placeholder.

Reply:
The copy document does not specify a link for this option:
https://docs.google.com/spreadsheets/d/1DtLOInAdlISKNOBm648vNmlfxoOBeHqDIJfsRKJddS8/edit#gid=0

Reply:
From Josh:
> Ignore. Follow the content in this spreadsheet. Use figma for design reference only